### ISSUE TITLE
Patching failed

### ISSUE THREAD

**Issue by dpo** [staticfloat/homebrew-julia#163]
```
==> Patching
patching file deps/Makefile
Hunk #1 succeeded at 1398 (offset 744 lines).
patching file Makefile
Hunk #1 FAILED at 236.
1 out of 1 hunk FAILED -- saving rejects to file Makefile.rej
```

**Comment by ZacCranko** [comment]
Got this same error here

**Comment by staticfloat** [comment]
This should be fixed by Zac's PR #165.
